sd-warning/cloudformation/sd-warning.yaml

```diff
@@ -0,0 +1,185 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Run minimal Onion site for The Guardian
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Networking
+        Parameters:
+          - VpcId
+          - InstanceSubnet
+      - Label:
+          default: AWS Configuration
+        Parameters:
+          - DataBucketName
+          - AMI
+          - Stage
+
+Mappings:
+  Constants:
+    Stack:
+      Value: editorial-systems-development
+    App:
+      Value: sd-warning
+
+Parameters:
+  VpcId:
+    Description: ID of the VPC onto which to launch the stack
+    Type: AWS::EC2::VPC::Id
+  InstanceSubnet:
+    Description: Subnet in which the instance will run
+    Type: AWS::EC2::Subnet::Id
+
+  DataBucketName:
+    Description: Name of the bucket that holds the static site and hidden_service config
+    Type: String
+
+  AMI:
+    Description: Base AMI for the instance
+    Type: AWS::EC2::Image::Id
+
+  Stage:
+    Description: Application stage
+    Type: String
+    AllowedValues:
+      - PROD
+      - CODE
+      - DEV
+    Default: PROD
+
+Resources:
+  InstanceSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Allow HTTP and HTTPS inbound connections
+      VpcId:
+        Ref: VpcId
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 8080
+          ToPort: 8080
+          CidrIp: 127.0.0.1/0
```

Comment:
I am not sure if we need this ingress rule? localhost traffic should not be affected by a security group